src/applications/simple-forms/21-10210/sass/10210-lay-witness-statement.scss

```diff
@@ -83,3 +83,16 @@
   font-size: 1.6rem !important;
   font-weight: 400 !important;
 }
+
+.form-review-panel[data-chapter="claimantPersonalInfoChapter"],
+.form-review-panel[data-chapter="claimantIdInfoChapter"],
+.form-review-panel[data-chapter="claimantAddrInfoChapter"],
+.form-review-panel[data-chapter="claimantContactInfoChapter"],
+.form-review-panel[data-chapter="veteranPersonalInfoChapter"],
+.form-review-panel[data-chapter="veteranIdentificationInfo"],
+.form-review-panel[data-chapter="veteranMailingAddressInfo"],
+.form-review-panel[data-chapter="veteranContactInfo"] {
+  .form-review-panel-page-header {
+    display: none;
```

Comment:
Are these here to hide the `<h4>`s? If so, are we sure that is still the desired outcome? The way @jeana-adhoc and I had worked it out on the other form, we kept the `<h4>`s because now they are unique and meaningful.

Reply:
> Are these here to hide the `<h4>`s? If so, are we sure that is still the desired outcome? The way @jeana-adhoc and I had worked it out on the other form, we kept the `<h4>`s because now they are unique and meaningful.

These specific `<h4>`s are hidden because they're basically just repeating the single-page chapter-titles (the parent `<h3>`s), that's why they're visually hidden.  If we didn't add the page-titles back for these chapters, then we'd have a bunch of Edit-buttons with just "Edit " for aria-label.

Reply:
Yep, and we've established that forms-lib will auto-hide the page-title IF it matches the chapter-title EXACTLY.  Since mine didn't, they had to be hidden via styling.